checkEligibility.js

```diff
@@ -19,12 +19,24 @@ async function patronCanPlaceTestHold (patronId, attempt = 1) {
     logger.error('Error: Placing a test hold on a test item did not generate an error!')
     return false
   } catch (e) {
-    patronHoldsPossible = e.response.data.description === 'XCirc error : Bib record cannot be loaded'
-    if (patronHoldsPossible) {
-      response = e.response.data
-      return true
+    // catch empty response from Sierra
+    if (!e.response) {
+      // don't want to try post requests more than once
+      if (!secondAttempt) {
+        logger.debug('Retrying patronCanPlacTestHold - empty Sierra response')
```

Comment:
Can we make this a `logger.info`. The logic looks correct, but I'd love to see evidence of it working via the logs after deployment.